### DISCUSSION TITLE
Is there any method to convert bbox to oobb?

### DISCUSSION THREAD

**Discussion by Snailya** [iTwin/itwinjs-core#4335] [Q&A] [answered]
I need the minimum bounding box to get the height, width and length of the box that encapsulates the element, so that I could get the ratio of volume to bounding box volumes and then use it with other properties to recongize the model that the element represents. But I'm confused with the 3d geometry concepts.

Given a bounding box point values and angles and origin point from placement, is the bounding box point value get's from a coordinates that uses the origin point in world coordinates as a new coordinates's origin? and the new coordinates' axies in the direction that angles represents?

**Comment by pmconne** [comment, marked as answer]
The [placement](https://www.itwinjs.org/reference/core-common/geometry/placement3d/)'s bounding box **is** an [OOBB](https://www.itwinjs.org/reference/core-common/geometry/elementalignedbox3d/). The placement's origin and angles tell how to position the element in world coordinates. The placement's `calculateRange` function produces an AABB from the OOBB.

If this doesn't resolve your question please state your question more clearly.
